ufm-test-services/metamask/package.json

```diff
@@ -25,5 +25,9 @@
     "static-server": "^2.2.1",
     "typescript": "^5.1.6",
     "viem": "^1.10.8"
+  },
+  "dependencies": {
+    "node-fetch": "^3.3.2",
```

Comment:
what version of node are we running that we need node-fetch?

Reply:
We could use `--experimental-fetch` in node 16. I believe @tynes has plans to upgrade us to node 18 in near future (if it hasn't already happened)

Reply:
Previously the monorepo's `.nvmrc` was using `v16`, but it's now using `v18` - I'll update and remove `node-fetch`